### PR TITLE
docs: Order menuweight for Docs ENT website 

### PR DESCRIPTION
**Pull request by anithamuthu2022**
This PR contains the following: 

- Edited the [Embedded JavaScript docs home page template](https://github.com/heartexlabs/label-studio/edit/ht/add-new-theme/docs/themes/v2/layout/home.ejs) with links for each card and modified the text to meet the user journey. 
- Ordered the pages and added section fields in the frontmatter until `import and data`.
- Reformatted `note` message.

### CODE REVIEW

**Review by makseq — APPROVED** [review]

File: docs/source/guide/release_notes.md [hunk 1]
```diff
@@ -3,9 +3,8 @@ title: Release notes for Label Studio Enterprise
 short: Release notes
 badge: <i class='ent'/></i>
 type: guide
-tier: all
-order: 120
-section: "Install"
+tier:
```
Comment (huguestennier) [inline docs/source/guide/release_notes.md:6]: this should have a value

File: docs/source/guide/release_notes.md [hunk 2]
```diff
@@ -3,9 +3,8 @@ title: Release notes for Label Studio Enterprise
 short: Release notes
 badge: <i class='ent'/></i>
 type: guide
-tier: all
-order: 120
-section: "Install"
+tier: 
+order:
```
Comment (huguestennier) [inline docs/source/guide/release_notes.md:7]: Maybe it works with a number here, but you should add one just in case

**Review by huguestennier — APPROVED** [review]
Just one file that needs attention, rest looks good

**Comment by anithamuthu2022** [comment]
https://user-images.githubusercontent.com/107003313/202618340-246f5a37-84de-4b97-b6db-0034ab54e4fc.mov

I see the left nav is not fully ordered as expected. @makseq you had mentioned in Slack that you will take the remaining parts of the left nav to order and add the section. Is that missed out in this PR.

File: docs/source/guide/release_notes.md (hunk 2 above)
Comment (anithamuthu2022) [reply docs/source/guide/release_notes.md:7]: Screen Shot 2022-11-17 at 8 45 25 PM

**Comment by anithamuthu2022** [comment]
The following files will not appear in the docs suite 

decommission_minio.md
install_requirements.md
label_studio_compare.md 
pingone.md
timeseries_video.md

**Pull request merged**